nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/parameter-contexts/ui/parameter-context-listing/edit-parameter-context/edit-parameter-context.component.html

```diff
@@ -26,12 +27,21 @@ <h2 mat-dialog-title>{{ this.isNew ? 'Add' : 'Edit' }} Parameter Context</h2>
                         @for (updateStep of requestEntity.request.updateSteps; track updateStep) {
                             <div class="flex justify-between items-center">
                                 <div class="value">{{ updateStep.description }}</div>
-                                @if (updateStep.complete) {
-                                    <div class="fa fa-check complete"></div>
+                                @if (updateStep.failureReason) {
+                                    <div class="fa fa-times text-red-400"></div>
```

Comment:
This should be using a class that derives its color from the theming system. It should probably be the default color from the warn palette.